### DISCUSSION TITLE
Dark Mode with Angular

### DISCUSSION THREAD

**Discussion by Flashtheorie** [tailwindlabs/tailwindcss#8182] [Help] [answered]
Hello everyone, I recently started to develop a project in Angular and Tailwind as a framework CSS and what I'm usually doing to toggle dark mode doesn't seem to work.

Usually, adding the class "dark" to the balise HTML works, but not with angular, 
anyone got any ideas ?

```
<!doctype html>
<html lang="fr" class="dark">
<head>
  <meta charset="utf-8">
  <title>Cv</title>
  <base href="/">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="icon" type="image/x-icon" href="favicon.ico">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/1.6.2/tailwind.min.css"/>
  <script src="https://kit.fontawesome.com/a513ca5fbe.js" crossorigin="anonymous"></script>
</head>
<body class="dark:bg-red-600">
 
  <app-root></app-root>
  
</body>
</html>
```

**Comment by Gelbotron** [comment, marked as answer]
Judging from the CDN link you are loading version 1.6.2. which doesn't support dark mode. 

If you want use Tailwinds dark-mode features from CDN, you should use at least Tailwind 3. 
This is done a little differently than before, as it is loaded as a `<script>` instead of `<link>`. 
https://tailwindcss.com/docs/installation/play-cdn

Keep in mind that this isn't supposed to be used in production, you will be better off with checking the Angular installation guide
https://tailwindcss.com/docs/guides/angular

**Reply by Flashtheorie** [reply]
Hello there, thank you for your reply, no, I've changed it a few days ago and it still not working.
`<script src="https://cdn.tailwindcss.com"></script>`

Edit 1 : Although I did install it as supposed to with Angular, thank you

Edit 2 : And it did work when adding the `darkMode: 'class',`into tailwind.config.js, thanks a lot !
